data-studio/data-studio.md

```diff
@@ -0,0 +1,105 @@
+## Create reports and dashboards with Data Studio
+
+![](images/example_dashboard.png)
+
+This is a short tutorial on how to set up a dashboard (or report as stated by
+data studio) in Google Data Studio (this can easily be done in other tools such
+as [Tableu](https://www.tableau.com/)).
+
+The aim is to show the potential of what you can do with the data from our
+Wittra sensor network.
+
+### Prerequisits
+
+In order to produce the dashboard it assumes you have already set up a
+database, to store the streamed data from the network. In this example we use
+[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enerprise
+data warehouse by Google.
+
+A Google account, to log into Data Studio. Note that Data Studio is free, but
+the queries to the data base may cost money depending on the database ad the
+amount of data.
+
+### 1. Create a new dashboard
+
+Start by logging into
+[Data Studio](https://datastudio.google.com/navigation/reporting) and click the
+Blank Report button.
+
+![](images/datastudio_start.png)
+
+### 2. Add a data source
+
+To populate the dashboard you will need to add a *connector* (in this case your
+BigQuery). When creating a new report the `add data view` will be opened
+automatically, if not click `add data` to the right in the top panel.
+
+![](images/datastudio_add_data.png)
+
+In this example we will create a dashboard for a specific tag showing
+temperature and usage. We will choose **BigQuery** as data source and retrieve
+the data with a **custom query**.
+
+```
+SELECT
+  *
+FROM
+  `<project-id.dataset.table>`
+where tag_id like '<tag-id>'
+order by timestamp desc
```

Comment:
Not an SQL expert but I would have the SQL keywords in caps, something like this:
```suggestion
SELECT *
FROM <project-id.dataset.table>
WHERE tag_id like <tag-id>
ORDER BY timestamp DESC
```
How would you add a "link" to a database outside the Google echosystem? The `<project-id.dataset.table>` assumes that your account can access the GCP project directly?
Then maybe add something like: "Replace `<tag-id>` with one of the device IDs in your project."